libmscore/property.cpp

```diff
@@ -78,6 +78,7 @@ static constexpr PropertyMetaData propertyList[] = {
       { Pid::DIRECTION,               false, "direction",             P_TYPE::DIRECTION,           DUMMY_QT_TRANSLATE_NOOP("propertyName", "direction")        },
       { Pid::STEM_DIRECTION,          false, "StemDirection",         P_TYPE::DIRECTION,           DUMMY_QT_TRANSLATE_NOOP("propertyName", "stem direction")   },
       { Pid::NO_STEM,                 false, "noStem",                P_TYPE::INT,                 DUMMY_QT_TRANSLATE_NOOP("propertyName", "no stem")          },
+      { Pid::HOOK_REVERSED,           false, "hookReverse",           P_TYPE::BOOL,                DUMMY_QT_TRANSLATE_NOOP("propertyName", "reverse hook")     },
```

Comment:
Probably better "hookReversed" or even "reversedHook" (requires changes elsewhere too) and "reversed hook"